src/lib/swapper/swappers/ThorchainSwapper/getThorTradeQuote/getTradeQuote.ts

```diff
@@ -125,45 +142,27 @@ export const getThorTradeQuote = async (
     )
   }
 
-  const thorSwapStreamingSwaps = getConfig().REACT_APP_FEATURE_THOR_SWAP_STREAMING_SWAPS
+  const getRouteValues = (quote: ThornodeQuoteResponseSuccess, isStreaming: boolean) => ({
+    source: isStreaming ? THORCHAIN_STREAM_SWAP_SOURCE : SwapperName.Thorchain,
+    quote,
+    // expected receive amount after slippage (no affiliate_fee or liquidity_fee taken out of this value)
+    // TODO: slippage is currently being applied on expected_amount_out which is emit_asset - outbound_fee,
```

Comment:
added a clarification comment in regards to how slippage_bps should be applied to the swap. i feel like the slippage_bps would be applied on the emit_asset, but I don't know for sure. the current calculation will be slightly over optimistic at worse